systemtest/src/main/java/io/strimzi/systemtest/utils/specific/KeycloakUtils.java

```diff
@@ -53,6 +56,30 @@ public static void deleteKeycloak(final String deploymentNamespace, final String
         Exec.exec(Level.INFO, "/bin/bash", PATH_TO_KEYCLOAK_TEARDOWN_SCRIPT, deploymentNamespace, getValidKeycloakVersion(), watchNamespace);
     }
 
+    public static void deployPostgres(final String deploymentNamespace) {
+        deployPostgres(deploymentNamespace, "main");
+    }
```

Comment:
I don't think that we want to use `main` branch. Also, why you need it, when you don't use it in tests?

Reply:
I think that we want to use `main`, as another one is `devel` ;-)
Because it's used by deafult as positional argument in prepare script.

Reply:
So postgres has just `main` and `devel` branches? No versions?
It's default argument, but you are not changing it in any test, anywhere. That doesn't make sense FMPOV.

Reply:
`It's default argument, but you are not changing it in any test, anywhere.`

Correct not changing yet. That can change, so it's prepared for future.

Reply:
Ok, method removed.